src/prefect/utilities/storage.py

```diff
@@ -0,0 +1,35 @@
+from typing import TYPE_CHECKING
+
+import prefect
+
+if TYPE_CHECKING:
+    from prefect.core.flow import Flow  # pylint: disable=W0611
+
+
+def get_flow_image(flow: "Flow") -> str:
```

Comment:
Isn't this a duplicate of https://github.com/PrefectHQ/prefect/blob/master/src/prefect/utilities/agent.py ?  We should consolidate

Reply:
Same logic but this one accepts a Flow object and the one that the agent uses doesn't ever have access to the flow object so it uses a GraphQLResult of a FlowRun